recipes/pcraster/meta.yaml

```diff
@@ -0,0 +1,101 @@
+{% set version = "4.3.0" %}
+
+package:
+  name: pcraster
+  version: {{ version }}
+
+source:
+  url: http://pcraster.geo.uu.nl/pcraster/packages/src/pcraster-{{ version }}.tar.bz2
+  sha256: fe346420f48aa7fd2870d5233b461ddc12ac965aa4cff0d471fbe9dbc4939232
+
+
+build:
+  number: 0
+  skip: True                                # [py2k]
+
+
+  missing_dso_whitelist:
+    - C:/Windows/System32/MSVCP140.dll      # [win]
+    - C:/Windows/System32/VCRUNTIME140.dll  # [win]
+    - C:/Windows/System32/imagehlp.dll      # [win]
+    - C:/Windows/System32/GLU32.dll         # [win]
+    - C:/Windows/System32/OPENGL32.dll      # [win]
+    - C:/Windows/System32/dwmapi.dll        # [win]
+    - C:/Windows/System32/WTSAPI32.dll      # [win]
+    - Library/bin/MSVCP140.dll              # [win]
+    - Library/bin/VCRUNTIME140.dll          # [win]
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}         # [linux]
+    - {{ cdt('mesa-dri-drivers') }}         # [linux]
+    - {{ cdt('libselinux') }}               # [linux]
+    - {{ cdt('libxdamage') }}               # [linux]
+    - {{ cdt('libxxf86vm') }}               # [linux]
+    - cmake >=3.15
+    - ninja
+    - make                                  # [win]
+
+  host:
+    - xorg-libxfixes                        # [linux]
+    - boost-cpp
+    - xerces-c
+    - qt
+    - ncurses                               # [not win]
+    - gdal
+    - libgdal
```

Comment:
Do you need both gdal and libgdl?  CC @conda-forge/gdal

Reply:
fyi, I did some more local builds on Linux, with meta.yaml as above and

```
conda-build . --channel conda-forge --override-channels --error-overlinking
```

a build fails, when adding ' --numpy 1.14' a build succeeds. When I remove gdal from host I can build without specifying the numpy 1.14 version.
I'll try the updated meta here